cluster/pre-existing/util.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Yes, the function create-certs in start-kubemark.sh needs to be overwritten.  Instead of going out to the common script for its definition  I define it here.